lambda/health_package/metric-settings.json

```diff
@@ -46,38 +248,410 @@
+    "EvaluationPeriods": 2,
+    "Period": 300,
+    "Multiplier": 1,
+    "Minimum": 600000,
+    "Description": "The number of bytes delivered to Splunk over the specified time period"
+  },
+  {
+    "Namespace": "AWS/Firehose",
+    "MetricName": "DeliveryToSplunk.DataAckLatency",
+    "Statistic": "Maximum",
+    "ComparisonOperator": "GreaterThanOrEqualToThreshold",
+    "EvaluationPeriods": 2,
+    "Period": 300,
+    "Multiplier": 1,
+    "Minimum": 50,
+    "Maximum": 95,
+    "Description": "The approximate duration (seconds) it takes to receive an acknowledgement from Splunk after Kinesis Data Firehose sends it data. The increasing or decreasing trend for this metric is more useful than the absolute approximate value. Increasing trends can indicate slower indexing and acknowledgement rates from Splunk indexers."
+  },
+  {
+    "Namespace": "AWS/Firehose",
+    "MetricName": "DeliveryToSplunk.DataFreshness",
     "Statistic": "Maximum",
     "ComparisonOperator": "GreaterThanOrEqualToThreshold",
     "EvaluationPeriods": 2,
     "Period": 300,
+    "Multiplier": 1,
+    "Minimum": 68,
+    "Maximum": 180,
+    "Description": "Age in seconds (from getting into Kinesis Data Firehose to now) of the oldest record in Kinesis Data Firehose. Any record older than this age has been delivered to Splunk"
+  },
+  {
+    "Namespace": "AWS/Firehose",
+    "MetricName": "DeliveryToSplunk.Records",
+    "Statistic": "Minimum",
+    "ComparisonOperator": "LessThanThreshold",
+    "EvaluationPeriods": 2,
+    "Period": 300,
+    "Multiplier": 1,
+    "Minimum": 1600,
+    "Description": "The number of records delivered to Splunk over the specified time period."
+  },
+  {
+    "Namespace": "AWS/Firehose",
+    "MetricName": "DescribeDeliveryStream.Latency",
+    "Statistic": "Maximum",
+    "ComparisonOperator": "GreaterThanOrEqualToThreshold",
+    "EvaluationPeriods": 2,
+    "Period": 300,
+    "Multiplier": 1,
+    "Minimum": 1,
+    "Maximum": 60,
+    "Description": "The time taken per DescribeDeliveryStream operation, measured over the specified time period."
+  },
+  {
+    "Namespace": "AWS/Firehose",
+    "MetricName": "DescribeDeliveryStream.Requests",
+    "Statistic": "Minimum",
+    "ComparisonOperator": "LessThanThreshold",
+    "EvaluationPeriods": 2,
+    "Period": 300,
+    "Multiplier": 1,
+    "Minimum": 38500,
+    "Description": "The number of bytes ingested successfully into the delivery stream over the specified time period after throttling."
+  },
+  {
+    "Namespace": "AWS/Firehose",
+    "MetricName": "IncomingPutRequests",
+    "Statistic": "Minimum",
+    "ComparisonOperator": "LessThanThreshold",
+    "EvaluationPeriods": 2,
+    "Period": 300,
+    "Multiplier": 1,
+    "Minimum": 0.99,
+    "Description": "The number of successful PutRecord and PutRecordBatch requests over the specified period of time after throttling."
+  },
+  {
+    "Namespace": "AWS/Firehose",
+    "MetricName": "IncomingRecords",
```

Comment:
IncomingRecords is a number of records. Why is the maximum based on half an hour in seconds?

Reply:
Removed